hello-world/host/src/test/java/com/r3/conclave/sample/host/NativeTest.java

```diff
@@ -0,0 +1,61 @@
+package com.r3.conclave.sample.host;
+
+import com.r3.conclave.host.AttestationParameters;
+import com.r3.conclave.host.EnclaveHost;
+import com.r3.conclave.host.EnclaveLoadException;
+import com.r3.conclave.host.MailCommand;
+import com.r3.conclave.mail.MailDecryptionException;
+import com.r3.conclave.mail.PostOffice;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledOnOs;
+import org.junit.jupiter.api.condition.OS;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+import java.nio.file.Paths;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * This test assumes a Linux platform and will try to load the enclave for real, either in simulation mode, or
+ * when compiled with the correct -PenclaveMode= flag, in debug or release mode.
+ */
+@EnabledOnOs(OS.LINUX)
+public class NativeTest {
+    // We start by loading the enclave using EnclaveHost, and passing the class name of the Enclave subclass
+    // that we defined in our enclave module.
```

Comment:
I don't think we do this below. I.e. we do not pass the Enclave subclass name. This is a recent update I think, the class name is found automatically.